### PR TITLE
PTRENG-3208 [Openshift] Artifactory 7.27.9 & Xray 3.35.0

### PR DESCRIPTION
**Pull request by giri-vsr**
#### PR Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [ ] Title of the PR starts with installer/product name (e.g. `[ansible/artifactory]`)
- [ ] CHANGELOG.md updated
- [ ] Variables and other changes are documented in the README.md

**What this PR does / why we need it**:

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

### CODE REVIEW

File: Openshift4/helm/openshift-artifactory-ha/CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,9 @@
 # JFrog  Openshift Artifactory-ha Chart Changelog
 All changes to this chart will be documented in this file.
 
+## [107.27.9] -  Dec15, 2021
```
Comment (alexhung) [inline Openshift4/helm/openshift-artifactory-ha/CHANGELOG.md:4]: Is the date here related to an older release? Shouldn't this be today or in near future?

File: Openshift4/helm/openshift-xray/CHANGELOG.md [hunk 2]
```diff
@@ -1,6 +1,9 @@
 # JFrog  Openshift Artifactory-Xray Chart Changelog
 All changes to this chart will be documented in this file.
 
+## [103.35.0] Dec 12, 2021
```
Comment (alexhung) [inline Openshift4/helm/openshift-xray/CHANGELOG.md:4]: Same here with the date

**Review by alexhung — COMMENTED** [review]
Look ok to me. Just a question on the version dates for all the charts.

File: Openshift4/helm/openshift-artifactory-ha/CHANGELOG.md (hunk 1 above)
Comment (giri-vsr) [reply Openshift4/helm/openshift-artifactory-ha/CHANGELOG.md:4]: This is the date operator image is published to redhat registry.Actual release to redhat operatorhub catalog which happens through bundle image is from different repo https://github.com/redhat-openshift-ecosystem/certified-operators

**Review by alexhung — APPROVED** [review]
:shipit:

**Pull request merged**